packages/lit-dev-content/site/docs/v3/components/decorators.md

```diff
@@ -44,9 +33,19 @@ The `@customElement` decorator defines a custom element, equivalent to calling:
 customElements.define('my-element', MyElement);
 ```
 
-The `@property` decorator declares a reactive property.
+The `@property` decorator declares a [reactive property](/docs/v3/components/properties/).
```

Comment:
I think we could tighten up the explanation visually by not using a code block:

```suggestion
The `@customElement` decorator defines a custom element, equivalent to calling `customElements.define('my-element', MyElement)`, while the `@property` decorator declares a [reactive property](/docs/v3/components/properties/).
```

Reply:
Actually not a fan of how this is looking. It's basically the same as the sentence above the sample code, and the `customElements.define` line doesn't look great without syntax highlighting.


<img width="826" alt="image" src="https://github.com/lit/lit.dev/assets/40413829/162ed052-051b-45af-9ee6-377f6f1b21b6">

Reply:
Ah, thanks for posting the image. I think this makes it clear that the explanatory text after the sample isn't necessary. We can just link "define a reactive property" and "register a custom element" to the right docs pages. Brevity for the win.

Reply:
Good point. The list of built-in has a short link that points up to this sample but it would then make more sense for it to point to http://lit.dev/docs/components/defining/

Reply:
yeah, that's the right page